e2e-tests/demand-backup-cloud/conf/demand-backup-cloud.yml

```diff
@@ -0,0 +1,92 @@
+apiVersion: pxc.percona.com/v1-6-0
+kind: PerconaXtraDBCluster
```

Comment:
and here

Reply:
Just copied it from the old test, it's not used since test parameters will override it.